Index: src/test/java/com/dongnao/weixinapp/repository/SellerInfoRepositoryTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/java/com/dongnao/weixinapp/repository/SellerInfoRepositoryTest.java	(date 1535265703770)
+++ src/test/java/com/dongnao/weixinapp/repository/SellerInfoRepositoryTest.java	(date 1535265703770)
@@ -0,0 +1,35 @@
+package com.dongnao.weixinapp.repository;
+
+import com.dongnao.weixinapp.dataobject.SellerInfo;
+import com.dongnao.weixinapp.utils.KeyUtil;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.junit.Assert.*;
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class SellerInfoRepositoryTest {
+    @Autowired
+    private SellerInfoRepository repository;
+    @Test
+    public void save() {
+        SellerInfo sellerInfo = new SellerInfo();
+        sellerInfo.setSellerId(KeyUtil.genUniqueKey());
+        sellerInfo.setUsername("admin");
+        sellerInfo.setPassword("admin");
+        sellerInfo.setOpenid("abc");
+        SellerInfo result = repository.save(sellerInfo);
+        Assert.assertNotNull(result);
+    }
+
+    @Test
+    public void findByOpenid() throws Exception {
+        SellerInfo result = repository.findByOpenid("abc");
+        Assert.assertEquals("abc", result.getOpenid());
+    }
+
+}
\ No newline at end of file
Index: src/main/java/com/dongnao/weixinapp/repository/SellerInfoRepository.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/dongnao/weixinapp/repository/SellerInfoRepository.java	(date 1535265605576)
+++ src/main/java/com/dongnao/weixinapp/repository/SellerInfoRepository.java	(date 1535265605576)
@@ -0,0 +1,12 @@
+package com.dongnao.weixinapp.repository;
+
+import com.dongnao.weixinapp.dataobject.SellerInfo;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+/**
+ * Created by 廖师兄
+ * 2017-07-23 23:04
+ */
+public interface SellerInfoRepository extends JpaRepository<SellerInfo, String> {
+    SellerInfo findByOpenid(String openid);
+}
Index: src/main/java/com/dongnao/weixinapp/service/iml/OrderServiceImpl.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/dongnao/weixinapp/service/iml/OrderServiceImpl.java	(revision 48f494f352ab51933b5ed83360b39bf5691ff1bb)
+++ src/main/java/com/dongnao/weixinapp/service/iml/OrderServiceImpl.java	(date 1535252328643)
@@ -179,6 +179,10 @@
 
     @Override
     public Page<OrderDTO> findList(Pageable pageable) {
-        return null;
+        Page<OrderMaster> orderMasterPage = orderMasterRepository.findAll(pageable);
+
+        List<OrderDTO> orderDTOList = OrderMaster2OrderDTOConverter.convert(orderMasterPage.getContent());
+
+        return new PageImpl<>(orderDTOList, pageable, orderMasterPage.getTotalElements());
     }
 }
Index: src/main/resources/templates.order/list.fxml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/resources/templates.order/list.fxml	(date 1535256693908)
+++ src/main/resources/templates.order/list.fxml	(date 1535256693908)
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import java.lang.*?>
+<?import java.util.*?>
+<?import javafx.scene.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+
+<AnchorPane xmlns="http://javafx.com/javafx"
+            xmlns:fx="http://javafx.com/fxml"
+            fx:controller="templates.order.List"
+            prefHeight="400.0" prefWidth="600.0">
+
+</AnchorPane>
Index: src/main/java/com/dongnao/weixinapp/service/iml/SellerServiceImpl.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/dongnao/weixinapp/service/iml/SellerServiceImpl.java	(date 1535265946458)
+++ src/main/java/com/dongnao/weixinapp/service/iml/SellerServiceImpl.java	(date 1535265946458)
@@ -0,0 +1,23 @@
+package com.dongnao.weixinapp.service.iml;
+
+import com.dongnao.weixinapp.dataobject.SellerInfo;
+import com.dongnao.weixinapp.repository.SellerInfoRepository;
+import com.dongnao.weixinapp.service.SellerService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+/**
+ * Created by 廖师兄
+ * 2017-07-29 23:15
+ */
+@Service
+public class SellerServiceImpl implements SellerService {
+
+    @Autowired
+    private SellerInfoRepository repository;
+
+    @Override
+    public SellerInfo findSellerInfoByOpenid(String openid) {
+        return repository.findByOpenid(openid);
+    }
+}
Index: src/main/java/com/dongnao/weixinapp/control/SellerOrderController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/dongnao/weixinapp/control/SellerOrderController.java	(date 1535256757973)
+++ src/main/java/com/dongnao/weixinapp/control/SellerOrderController.java	(date 1535256757973)
@@ -0,0 +1,33 @@
+package com.dongnao.weixinapp.control;
+
+import com.dongnao.weixinapp.dto.OrderDTO;
+import com.dongnao.weixinapp.service.OrderService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.servlet.ModelAndView;
+
+import java.util.Map;
+
+@Controller
+@RequestMapping("/seller/order")
+public class SellerOrderController  {
+    @Autowired
+    private OrderService orderService;
+    @GetMapping("/list")
+    public ModelAndView list(@RequestParam(value = "page", defaultValue = "1") Integer page,
+                             @RequestParam(value = "size", defaultValue = "10") Integer size,
+                             Map<String, Object> map) {
+        PageRequest request = new PageRequest(page - 1, size);
+        Page<OrderDTO> orderDTOPage = orderService.findList(request);
+        map.put(" ", orderDTOPage);
+        map.put("currentPage", page);
+        map.put("size", size);
+//        orderDTOPage.getTotalPages()
+        return new ModelAndView("order/list", map);
+    }
+}
Index: src/main/java/com/dongnao/weixinapp/service/OrderService.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/dongnao/weixinapp/service/OrderService.java	(revision 48f494f352ab51933b5ed83360b39bf5691ff1bb)
+++ src/main/java/com/dongnao/weixinapp/service/OrderService.java	(date 1535252302326)
@@ -4,6 +4,10 @@
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.Pageable;
 
+/**
+ * Created by 廖师兄
+ * 2017-06-11 18:23
+ */
 public interface OrderService {
 
     /** 创建订单. */
@@ -27,5 +31,4 @@
     /** 查询订单列表. */
     Page<OrderDTO> findList(Pageable pageable);
 
-
 }
Index: pom.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- pom.xml	(revision 48f494f352ab51933b5ed83360b39bf5691ff1bb)
+++ pom.xml	(date 1535278748668)
@@ -45,7 +45,12 @@
             <artifactId>gson</artifactId>
             <version>2.8.5</version>
         </dependency>
-
+        <dependency>
+            <groupId>redis.clients</groupId>
+            <artifactId>jedis</artifactId>
+            <version>2.1.0</version>
+            <scope>test</scope>
+        </dependency>
         <dependency>
             <groupId>org.mybatis.spring.boot</groupId>
             <artifactId>mybatis-spring-boot-starter</artifactId>
Index: src/main/java/com/dongnao/weixinapp/service/SellerService.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/dongnao/weixinapp/service/SellerService.java	(date 1535265949670)
+++ src/main/java/com/dongnao/weixinapp/service/SellerService.java	(date 1535265949670)
@@ -0,0 +1,19 @@
+package com.dongnao.weixinapp.service;
+
+
+import com.dongnao.weixinapp.dataobject.SellerInfo;
+
+/**
+ * 卖家端
+ * Created by 廖师兄
+ * 2017-07-29 23:14
+ */
+public interface SellerService {
+
+    /**
+     * 通过openid查询卖家端信息
+     * @param openid
+     * @return
+     */
+    SellerInfo findSellerInfoByOpenid(String openid);
+}
Index: src/main/resources/templates.order/list.ftl
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/resources/templates.order/list.ftl	(date 1535256774628)
+++ src/main/resources/templates.order/list.ftl	(date 1535256774628)
@@ -0,0 +1,1 @@
+<h1>${orderDTOPage.getTotalPages}</h1>
\ No newline at end of file
